### DISCUSSION TITLE
Trakt integration

### DISCUSSION THREAD

**Discussion by xiongster** [sct/overseerr#1560] [Feature Requests]
I dont know if its possible or not. Getting "seen" status synced from trakt to overseerr, so that users can know which movies have they already watched when looking for new movies.

**Comment by pchow98** [comment]
Also simkl.com integration would be great

**Comment by jpdsc** [comment]
Would be great. However, best would be to get the current Trakt lists from Radarr/Sonarr?

**Comment by JonnySiHI** [comment]
Trakt integration would be nice

**Comment by ericvlog** [comment]
hello, this game changer if can include in overseerr

**Comment by Azmarine13** [comment]
Any Update on this?
